swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0.0):
```

Comment:
We actually do not want to have category in the `__init__` function. We want the category to always be "Clothing", but if we put category here, the user can set category to whatever they want, which isn't good!